webpack.config.js

```diff
@@ -36,8 +36,10 @@ module.exports = {
     proxy: {
       "/api": "http://localhost:3000/",
       "/login": "http://localhost:3000/",
+      "/logout": "http://localhost:3000/",
       "/signup": "http://localhost:3000/",
-      "/boards": "http://localhost:3000/"
+      "/boards": "http://localhost:3000/",
+      "/sessionTest": "http://localhost:3000/",
     },
   },
```

Comment:
I just saw this in the webpack docs and wondering if this will work as well:

```
proxy: [
  {
    context: ['/api', '/login', '/logout', '/signup', '/boards', '/sessionTest'],
    target: http://localhost:3000,
  }
]
```

Reference: https://webpack.js.org/configuration/dev-server/#devserverproxy

Reply:
It seems to work!  Implemented